integration_tests/src/main/python/window_function_test.py

```diff
@@ -134,15 +134,15 @@
     ('a', IntegerGen()),
     ('b', LongGen(nullable=True))]
 
-part_and_order_gens = [long_gen, DoubleGen(no_nans=True, special_cases=[]),
+part_and_order_gens = [long_gen, DoubleGen(special_cases=[]),
```

Comment:
Curious why we still are avoiding special cases?  Feels like the we need to followup to understand why we're avoiding special cases in these tests.  That's a separate issue, not blocking for this PR.

Reply:
I did a quick check by removing the empty special_cases and it didn't result in failures. This could just be reminiscent of the past.   

I have created a follow-on [issue](https://github.com/NVIDIA/spark-rapids/issues/9338) to track it